src/components/StateResults.vue

```diff
@@ -1,56 +1,73 @@
 <template>
-  <div
-    :class="suit()"
-    v-if="state && state.results"
-  >
-    <slot v-bind="state.results">
-      <p>Use this component to have a different layout based on a certain state.</p>
-      <p>Fill in the slot, and get access to the following things on the <code>slot-scope</code>:</p>
-      <pre>{{ Object.keys(state.results) }}</pre>
+  <div :class="suit()" v-if="state && state.results">
+    <slot
+      v-bind="
+        Object.assign(
+          {},
+          // TODO: remove this spreading in the next major version
+          state.results,
+          { results: state.results, state: state.state }
+        )
+      "
+    >
+      <p>
+        Use this component to have a different layout based on a certain state.
+      </p>
+      <p>
+        Fill in the slot, and get access to the following things on the
+        <code>slot-scope</code>:
+      </p>
+      <pre>results: {{ Object.keys(state.results) }}</pre>
+      <pre>state: {{ Object.keys(state.state) }}</pre>
     </slot>
   </div>
 </template>
 
 <script>
-import { createSuitMixin } from '../mixins/suit';
-import { createWidgetMixin } from '../mixins/widget';
+import { createSuitMixin } from "../mixins/suit";
+import { createWidgetMixin } from "../mixins/widget";
 
 const connectStateResults = (renderFn, unmountFn) => (widgetParams = {}) => ({
   init({ instantSearchInstance }) {
     renderFn(
       {
+        state: undefined,
         results: undefined,
         instantSearchInstance,
-        widgetParams,
+        widgetParams
       },
       true
     );
   },
 
-  render({ results, instantSearchInstance }) {
+  render({ results, instantSearchInstance, state }) {
     const resultsCopy = Object.assign({}, results);
```

Comment:
let's find an alternative for shallow copying here, since it's not available in IE11

Reply:
Switch to Babel?